scripts/train.py

```diff
@@ -241,12 +242,18 @@ def dummy_init_fn(module: torch.nn.Module) -> None:
 
 
 if __name__ == "__main__":
-    mp.set_start_method("spawn")
+    try:
+        mp.set_start_method("spawn", force=True)
+    except RuntimeError as e:
+        print(f"failed to set multiprocessing start method: {e}")
```

Comment:
I did not know this could fail?

Reply:
Yea I saw this happen a couple times, not sure why exactly.